src/cornerstone/seoAssessor.js

```diff
@@ -33,75 +34,142 @@ const CornerstoneSEOAssessor = function( i18n, options ) {
 	Assessor.call( this, i18n, options );
 	this.type = "CornerstoneSEOAssessor";
 
-	this._assessments = [
-		new IntroductionKeywordAssessment(),
-		new KeyphraseLengthAssessment(),
-		new KeywordDensityAssessment(),
-		new MetaDescriptionKeywordAssessment(),
-		new MetaDescriptionLength( {
-			scores:	{
-				tooLong: 3,
-				tooShort: 3,
-			},
-		} ),
-		new SubheadingsKeyword(
-			{
+	if ( process.env.YOAST_RECALIBRATION === "enabled" ) {
```

Comment:
There is some repeated code in this function. Not every assessment gets replaced for recalibration (yet 😛 ).
Perhaps we can have a list with default assessments, and then an if-else to add only the assessments that are different (f you catch my drift). Would save some lines and make it more clear what assessments have changed 🙂 .

Reply:
Upon further investigation together with @hansjovis and @manuelaugustin it was decided to stick with the original version as it allows fast transition between beta-recalibration and forever-recalibration. Also, as per https://github.com/Yoast/YoastSEO.js/issues/1913#issuecomment-432659703 we should not be afraid of duplicate code.